### DISCUSSION TITLE
customizable project statistics

### DISCUSSION THREAD

**Discussion by kiwitwo** [ScratchAddons/ScratchAddons#5687] [Ideas] [answered]
seeing as the project statistics icons (love count, favorite count, remix count, and view count) have been updated recently to be more dark and outlined, i thought that now would be a good time to implement an addon to change the individual colors of these icons.

you'd be able to change the following values:

- love count color (unclicked)
- love count color (clicked)
- favorite count color (unclicked)
- favorite count color (clicked)
- remix count color
- view count color

this would presumably also have default presets (current default colors, old default colors, etc.)

**Comment by mxmou** [comment, marked as answer]
This would make it impossible to change the colors automatically depending on the background color selected in the website dark mode addon.
